cmd/junod/cmd/root.go

```diff
@@ -108,7 +107,7 @@ func initRootCmd(rootCmd *cobra.Command, encodingConfig params.EncodingConfig) {
 		AddGenesisWasmMsgCmd(app.DefaultNodeHome),
 		tmcli.NewCompletionCmd(rootCmd, true),
 		// testnetCmd(app.ModuleBasics, banktypes.GenesisBalancesIterator{}),
-		debug.Cmd(),
+		ConvertBech32Cmd(),
```

Comment:
Why is the `junod debug` command removed with this? This can still be added on top of the debug.Cmd()

Reply:
sensible!